src/main/kotlin/blackjack/ui/OutputView.kt

```diff
@@ -0,0 +1,29 @@
+package blackjack.ui
+
+import blackjack.domain.Player
+
+object OutputView {
+
+    fun showDistribution(players: List<Player>) {
+        val names = getNames(players)
+        println("$names 에게 2장의 카드를 나누었습니다.")
+        players.forEach {
+            showHand(it)
+        }
+    }
+
+    private fun getNames(players: List<Player>): String {
+        return players.joinToString(", ") { it.name }
```

Comment:
```suggestion
        return players.joinToString { it.name }
```
https://github.com/next-step/kotlin-blackjack/pull/215#discussion_r889536300